### ISSUE TITLE
Backspace also delete <br> tags

### ISSUE THREAD

**Issue by antonwebnauts** [froala/wysiwyg-editor#4480]
In editor we have just plain paragraph. Option enter is default(FroalaEditor.ENTER_P).  Use combination Shift+Enter to insert <br> tag. After try to delete text below enters with the backspace button

##### Expected behavior.
It wiil delete a  letter, which is located left of cursor 

##### Actual behavior.
It delete a letter and a <br> tag

##### Steps to reproduce the problem.
In editor we have just plain paragraph. Option enter is default(FroalaEditor.ENTER_P).  Use combination Shift+Enter to insert <br> tag. After try to delete text below enters with the backspace button
##### Editor version.
I used https://froala.com/wysiwyg-editor/examples/enter/ and in my version of Froala 4.0.11

##### OS.
Windows 10

##### Browser.
Google Chrome, Mozilla

##### Recording.

https://user-images.githubusercontent.com/81618890/178103213-733f0a47-c762-40b4-bf1b-92ec02681e46.mp4

**Comment by Lateworm** [comment]
I've now seen this problem reported separately from many users of my team's product.

@antonwebnauts's repro points out that this problem arises if Froala is configured to use `<br>`s. Regardless of 'enter' config, this problem is becoming very common for us when users paste in content. Also, even with default `enter` config that does not user `<br>`s, users can create this problem for themselves by using shift-enter.

**Comment by nategadzhi** [comment]
I don't think this depends on the operating system — quite easy to reproduce on Mac OS + Firefox, too.

As a work around, seems like if the <br> tags are wrapped in <p></p>, they don't get deleted. 

@antonwebnauts, what version of Froala are you running, and did you try previous versions?

**Comment by nategadzhi** [comment]
https://www.loom.com/share/8e729add420f463da393b2a78b032366 This is still going.

**Comment by chrille112** [comment]
We're experiencing this as well. We're running 4.0.14 via angular-froala-wysiwyg

**Comment by Sameera-Github** [comment]
Hi Team,
Please provide an ETR for the fix/ work around.
This issue is very annoying for the clients.
We are currently on Froala 4.0.14

**Comment by claudioldf** [comment]
any news about this? we are also experiencing this issue even using the licensed version
It seems issue was reported a couple of month ago, and it is very annoying for the clients :/

**Comment by timoteo27** [comment]
Version 4.0.15 seems to fix the issue 🙌🏼  https://github.com/froala/wysiwyg-editor/releases/tag/v4.0.15
> Fixed, BackSpace removes extra BR from editor content.

Note: Froala editor example isn't on the latest version (currently v4.0.14)

**Comment by zahidkarim24** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
